7/zts/vendor/karibbu/Dockerfile

```diff
@@ -8,6 +8,7 @@ request_order=GP\n\
 expose_php=Off\n\
 enable_dl=Off\n\
 short_open_tag=Off\n\
+zend.assertions=-1\n\
```

Comment:
We don't have to, it's a runtime setting in fact... so my point of view was to set `end.assertions=-1` only when run env is prod but in fact... you're right we should have prod settings by default and change them when we switch to dev or test.

We can simply use either `ini_set('zend.assertions', 1);` or 'php -d zend.assertions=1`.